eve/main.yml

```diff
@@ -30,3 +41,17 @@ stages:
         urls:
           - "*.iso"
           - "SHA256SUM"
+
+  lint:
+    worker:
+      type: kube_pod
+      path: eve/workers/pod-linter/pod.yaml
+      images:
+        docker-linter: eve/workers/pod-linter
+    steps:
+    - Git: *git_pull
+    - ShellCommand:
+        name: linting steps
```

Comment:
I think I read somewhere we prefer complete sentences, maybe `Run all linting steps`